assets/js/modules/tagmanager/components/setup/SetupForm.js

```diff
@@ -61,7 +60,6 @@ export default function SetupForm( { finishSetup } ) {
 	const { setValues } = useDispatch( CORE_FORMS );
 	const { activateModule } = useDispatch( CORE_MODULES );
 	const { submitChanges } = useDispatch( STORE_NAME );
-	const dispatchAnalytics = useDispatch( MODULES_ANALYTICS );
```

Comment:
:thinking:   I'm assuming this was left around from a previous change. But it's fine to remove